lib/utils.pm

```diff
@@ -333,7 +333,7 @@ sub workaround_type_encrypted_passphrase {
     return if get_var('UNENCRYPTED_BOOT');
     return if !get_var('ENCRYPT') && !get_var('FULL_LVM_ENCRYPT');
     # ppc64le on pre-storage-ng boot was part of encrypted LVM
-    return if !get_var('FULL_LVM_ENCRYPT') && !is_storage_ng && !check_var('ARCH', 'ppc64le');
+    return if !get_var('FULL_LVM_ENCRYPT') && !is_storage_ng && !get_var('OFW');
```

Comment:
Here the check is about PPC having a proposed separated **/boot** partition.
AFAIK OFW and PPC are not always the same. So here would make more sense to have

```perl
return if !get_var('FULL_LVM_ENCRYPT') && !is_storage_ng && get_var('ARCH) !~ /^ppc/;
```

Reply:
As per products/opensuse/templates  OFW is defined for ppc64 and ppc64le ARCH, so I consider the same for all the tests of this commit.
Why do you want to keep a difference between the two ?

Reply:
It is a coincidence that only our ppc machines have OFW. We could have another machines non-ppc which use OFW.